sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -589,6 +596,8 @@ def test_read_export_exception(self, exception_type, error_message):
           table='project:dataset.table',
           gcs_location="gs://temp_location")
 
+    p.run().wait_until_finish(duration=30)
```

Comment:
this will cause the pipeline to be ran twice (see https://stackoverflow.com/a/52282001/1255356)